<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.test.automation.testStartup.TestLauncher" tests="2" time="0.000" ignored="0" failures="0" timestamp="27 Nov 2017 17:03:54 GMT" skipped="1" hostname="Admins-MacBook-Pro.local" errors="1">
  <testcase name="executeTestCase" time="0.000" classname="com.test.automation.testStartup.TestLauncher">
    <skipped/>
  </testcase> <!-- executeTestCase -->
  <testcase name="beforeSuite" time="0.000" classname="com.test.automation.testStartup.TestLauncher">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.test.automation.utilities.FileUtils.getFileNameFromDirectory(FileUtils.java:37)
	at com.test.automation.utilities.FileUtils.getFileNameWithoutExtension(FileUtils.java:59)
	at com.test.automation.testStartup.TestLauncher.beforeSuite(TestLauncher.java:166)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeSuite -->
</testsuite> <!-- com.test.automation.testStartup.TestLauncher -->
